admin_guide/crictl.adoc

```diff
@@ -0,0 +1,482 @@
+[[Admin-crictl-node-debug-guide]]
+= Admin crictl node debug guide
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:toclevels: 5
+:prewrap!:
+:context: debugging-crio-containers-using-crictl
+
+toc::[]
+
+== Overview
+
+The *crictl* command line tool offers many powerful methods to debug pods, containers, and images.
+
+This page shows how *crictl* can be used in debugging pods and containers created by *OpenShift* when using *CRI-O* as the runtime.
+
+== Debugging with crictl
+
+Let's create a simple pod called *my-pod* running two containers using the *oc* CLI.
+----
+$ cat pod.yml 
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod
+spec:
+  containers:
+  - name: my-httpd
+    image: docker.io/library/httpd:2.4-alpine
+    ports:
+    - containerPort: 9876
+  - name: shell
+    image: docker.io/library/centos:7
+    tty: true
+    command:
+    - "bin/bash"
+    - "-c"
+    - "top"
+
+$ oc create -f pod.yml
+pod "my-pod" created
+
+$ oc get pods
+NAME      READY     STATUS    RESTARTS   AGE
+my-pod    2/2       Running   0          10s
+----
+
+Ensure that the _runtime-endpoint_ of *crictl* is pointing to the *CRI-O* socket in the config file */etc/crictl.yaml*.
+----
+$ cat /etc/crictl.yaml 
+runtime-endpoint: unix:///var/run/crio/crio.sock
+image-endpoint: ""
```

Comment:
This should also point to the crio socket.

Reply:
Fixed.